src/parser/priest/discipline/modules/shadowlands/covenants/MindGames.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+
+import ItemHealingDone from 'interface/ItemHealingDone';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+
+import SPELLS from 'common/SPELLS';
+import { formatNumber } from 'common/format';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+
+import AtonementDamageSource from '../../features/AtonementDamageSource';
+import isAtonement from '../../core/isAtonement';
+
+class Mindgames extends Analyzer {
+  static dependencies = {
+    atonementDamageSource: AtonementDamageSource,
+  };
+
+  atonementHealing = 0;
+  directHealing = 0;
+  preventedDamage = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasCovenant(COVENANTS.VENTHYR.id);
+
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER), this.onHeal);
+    this.addEventListener(Events.absorbed.by(SELECTED_PLAYER).spell(SPELLS.MINDGAMES_ABSORB), this.onMindgamesAbsorbed);
+
```

Comment:
Whitespace got a bit funky in the file with the last few commits; L33, L37 should be removed. Then good to go!